sleap/io/cameras.py

```diff
@@ -850,6 +860,22 @@ def update_points(
             exclude_complete: If True, then do not update points that are marked as
                 complete. Default is True.
         """
+        # Ensure we are working with a float array
+        points = points.astype(float)
+
+        # Check if points are 3D
+        is_3d = points.shape[-1] == 3
```

Comment:
THis would always evaluate to False at the moment since the `points` passed into `InstanceGroup.update_points` are:
```
points: Numpy array of shape (M, N, 2) where M is the number of views, N is
                the number of Nodes, and 2 is for x, y.
```
. We want the 3D points passed in here. That means we also want the 3D points passed into `FrameGroup.upsert_points`... which means we need to basically move code from both the following:
https://github.com/talmolab/sleap/blob/9775fc22d4917ba7a9b44ebfad45db790bc5d0c1/sleap/gui/commands.py#L3769-L3791
and from `FrameGroup.upsert_points` to `InstanceGroup.update_points`